app/models/event_type.rb

```diff
@@ -7,6 +7,14 @@ class EventType
       "School or University event" => 222_750_009,
     }.freeze
 
+  QUERY_PARAM_NAMES =
```

Comment:
Are these param names taken from the CRM?

I'm asking because

a) I don't think they are really as useful as they could be eg 'online_event' is misleading as it is only Online Q&A in reality but could be read as any online event

b) they are longer than needed which increases URL length therefore making it harder to read in GA and harder to get right in links in emails etc.

If we have flexibility, I would perhaps suggest something like

      "ttt" => 222_750_001,
      "tttqt" => 222_750_007,
      "onlineqa" => 222_750_008,
      "provider" => 222_750_009,
      
I've deliberately included 'ttt' in the question time events as it then is easy to query all ttt events in GA.

NB event types are likely to change in the next two terms anyway, but good to get some principles in place!

Reply:
The ones above ("School or University event" etc) _are_ from the CRM, but there's no reason for the `QUERY_PARAM_NAMES` to follow suit (other than consistency). I'll change them and add a comment so it's clear what's what